packages/trampoline/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@endo/trampoline",
+  "version": "0.1.0",
+  "description": "Multicolor trampolining for recursive operations",
+  "keywords": [
+    "trampoline",
+    "recursive",
+    "async",
+    "sync",
+    "generator"
+  ],
+  "author": "Endo contributors",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/endojs/endo/blob/master/packages/trampoline/README.md",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/endojs/endo.git",
+    "directory": "packages/trampoline"
+  },
+  "bugs": {
+    "url": "https://github.com/endojs/endo/issues"
+  },
+  "type": "module",
+  "main": "./index.js",
+  "module": "./index.js",
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "default": "./index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "types": "./index.d.ts",
+  "scripts": {
+    "build": "exit 0",
+    "build:types": "tsc --build tsconfig.build.json",
+    "clean:types": "git clean -f '*.d.ts*'",
+    "cover": "c8 ava",
+    "lint": "yarn lint:types && yarn lint:eslint",
+    "lint-fix": "eslint --fix .",
+    "lint:eslint": "eslint .",
+    "lint:types": "tsc",
+    "test:types": "yarn build:types && tsd -f \"test/*.test-d.ts\"; yarn clean:types",
+    "test:ava": "ava",
+    "test": "yarn test:types && yarn test:ava"
+  },
+  "devDependencies": {
+    "ava": "^6.1.2",
+    "babel-eslint": "^10.0.3",
+    "c8": "^7.14.0",
+    "eslint": "^8.57.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-eslint-comments": "^3.1.2",
+    "eslint-plugin-import": "^2.29.0",
+    "prettier": "^3.0.0",
+    "tsd": "^0.31.0",
+    "typescript": "~5.5.0-dev.20240327"
+  },
+  "files": [
+    "LICENSE*",
+    "SECURITY*",
+    "src",
+    "*.js",
+    "*.ts"
+  ],
+  "private": true,
+  "eslintConfig": {
+    "extends": [
+      "plugin:@endo/internal"
```

Comment:
I copy/pasted this. 

I don't understand why this is here and why we are pulling in all of those eslint dev deps.

Reply:
Also, I am not sure why we're pulling in all of these dev deps in _any_ workspace.  

Why not let them all live in the monorepo root?

Reply:
Let’s follow-up about this separately and solve it homogeneously if we can. The config for `ses` and below is a little different than above `ses`, where we can count on Hardened JavaScript for some invariants. But, otherwise, if we can hoist more, I’m in favor.

If we are going to use this package in SES, it requires the more-strict lint (no-polymorphic-call), and I assume you’re shooting for that. (to follow-up with a SES refactor). There’ll be a bit of duplication with uncurryThis and getting commons.

@naugtur did you review this with a close eye on whether this would interfere with Hermes?